### ISSUE TITLE
Group and user isolation for Webui/Sophomorix

### ISSUE THREAD

**Issue by kiarn** [linuxmuster/linuxmuster-base7#91]
Hello @HappyBasher,

In order to restrain the rights in the Webui, we need a sudoer file e.g. `/etc/sudoers.d/linuxmuster` template with content like : 

```
%DOMAIN\\all-teachers ALL=(ALL:ALL) NOPASSWD:ALL
%DOMAIN\\all-admins ALL=(ALL:ALL) NOPASSWD:ALL
%DOMAIN\\all-students ALL=(ALL:ALL) NOPASSWD: /usr/sbin/sophomorix-passwd,/usr/sbin/sophomorix-query
```

DOMAIN should be the samba domain on the server.
Sorry for the convenience, I didn't thought at this solution last week.

Thanks

Arnaud

**Comment by HappyBasher** [comment]
Fix published in linuxmuster-base7 7.0.57.

**Comment by kiarn** [comment]
Cool thanks @HappyBasher !
But I think it's missing a % before the domain name, at the beginning of the line, in order to identify a group and not an user :s 
Sorry ...

**Comment by HappyBasher** [comment]
Fixed at least in 7.0.58.
